Extension/package.json

```diff
@@ -42,6 +42,168 @@
+              "path": "walkthrough/build-project-windows.md"
+            }
+          },
+          {
+            "id": "run.project.mac",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-clang-mac#_debug-helloworldcpp)",
+            "when": "isMac",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on macOS",
+              "path": "walkthrough/run-and-debug-project-mac.md"
+            }
+          },
+          {
+            "id": "run.project.linux",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-linux#_debug-helloworldcpp)",
+            "when": "isLinux",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Linux",
+              "path": "walkthrough/run-and-debug-project-linux.md"
+            }
+          },
+          {
+            "id": "run.project.windows",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-msvc#_debug-helloworldcpp)",
+            "when": "isWindows",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Windows",
+              "path": "walkthrough/run-and-debug-project-windows.md"
+            }
+          },
+          {
+            "id": "install.cmake.tools",
+            "title": "Install CMake Tools",
+            "description": "Do you build your project with CMake? Install the CMake Tools extension to seamlessly build and debug CMake project. \n[Install CMake Tools](command:workbench.extensions.installExtension?%22ms-vscode.cmake-tools%22)",
+            "doneOn": {
+              "command": "workbench.extensions.installExtension"
+            },
+            "media": {
+              "altText": "screenshot of CMake Tools extension page",
+              "path": {
+                "light": "walkthrough/cmake-tools-light.png",
+                "dark": "walkthrough/cmake-tools-dark.png",
+                "hc": "walkthrough/cmake-tools-hc.png"
+              }
+            }
+          },
+          {
+            "id": "watch.tutorials",
+            "title": "Lean back and get started",
+            "description": "Watch this series of short and practical videos about setting up C++ IntelliSense, building, and debugging C++ projects. \n[Watch Tutorials](https://code.visualstudio.com/docs/cpp/introvideos-cpp)",
+            "media": {
+              "altText": "screenshot with preview of video tutorial and play button",
+              "path": "walkthrough/getting-started-video.PNG"
```

Comment:
The image has a play button, so I thought it was a video I could click but clicking it didn't work...not sure if users would hit that in the context of the walkthrough though.

Reply:
I did the same thing when going through VS Code's walkthrough with this same screenshot. I'm going to ask them if there's a way to make the image clickable.